.github/ISSUE_TEMPLATE/first_rc_checklist.md

```diff
@@ -22,6 +22,7 @@ labels: task
 * [ ] Build and upload conda packages on buildfarm (check "upload").
 * [ ] Build wheels (`$PYTHON_VERSIONS`) on the buildfarm.
 * [ ] Verify packages uploaded to Anaconda Cloud and move to `numba/label/main`.
+* [ ] Build sdist locally using `python setup.py sdist --user=ci --group=numba` with umask `0022`.
```

Comment:
Did the umask used get confirmed by @seibert ? xref: https://github.com/numba/numba/issues/7800#issuecomment-1024522300

Reply:
yes, this umask is correct